src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] I would advise you not to disable the Linter, You have to solve it manually.
- A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs

Reply:
thanks for pointing it out.

Reply:
fixed